common/traits/btr_species_traits.txt

```diff
@@ -820,15 +828,68 @@ btr_yahg_keen_mind = {
 	}
 
 	modifier = {
-		species_leader_exp_gain = 0.25
-		planet_jobs_engineering_research_produces_mult = 0.2
-		planet_jobs_physics_research_produces_mult = 0.2
-		planet_jobs_society_research_produces_mult = 0.2
-		intel_decryption_mult = 0.2
-		intel_encryption_mult = 0.2
+		pop_growth_speed = -0.15
+		planet_pops_organics_food_upkeep_mult = 0.15
+		pop_housing_usage_mult = 0.25
+		army_damage_mult = 0.66
+		army_morale = 0.66
```

Comment:
This is a lot, not that I hate it, but I am not sure of the lore reasons behind this decision, in the end yahg aren't that advanced.

Reply:
@puckhead73 Your input could be nice.

Reply:
@cemal-efe-g Not sure either although to be fair they are supposed to be biologically really powerful so idk I just followed the sheet. To be fair the geth(Nazara) have way crazier armies than this from what I have seen so far.

Reply:
@maxim-a-k Everyone seemed to be okay with this, think we can resolve or waiting on someone's input?

Reply:
Resolved in discord